content/sdk/10.js/00.ethers/10.v5/10.types.md

```diff
@@ -259,103 +530,193 @@ Deprecated in favor of `[LogProof](#logproof)`
 
 Type definition for a paymaster input specified as either approval based or general.
 
-- `ApprovalBasedPaymasterInput` | `GeneralPaymasterInput`
+::collapsible
+  ::field{name="ApprovalBasedPaymasterInput" type="ApprovalBasedPaymasterInput"}
+  ::
+  ::field{name="GeneralPaymasterInput" type="GeneralPaymasterInput"}
+  ::
+::
 
 ## `PaymasterParams`
 
 Type defining a paymaster by address and the bytestream input.
 
-- `paymaster`: `Address` - The address of the paymaster.
-- `paymasterInput`: `BytesLike` - The bytestream input for the paymaster.
+::collapsible
+  ::field{name="paymaster" type="Address"}
+  The address of the paymaster.
+  ::
+  ::field{name="paymasterInput" type="BytesLike"}
+  The bytestream input for
 
-## `PayloadSigner`
+ the paymaster.
+  ::
+::
 
-Signs various types of payloads, optionally using a some kind of secret.
+## `PayloadSigner`
 
-- `payload`: `BytesLike` - The payload that needs to be sign.
-- `secret`: `any` - The secret used for signing the `payload`.
-- `provider` `null | Provider` -  The provider is used to fetch data from the network if it is required for signing.
+Signs various types of payloads, optionally using some kind of secret.
+
+::collapsible
+  ::field{name="payload" type="BytesLike"}
+  The payload that needs to be signed.
+  ::
+  ::field{name="secret" type="any"}
+  The secret used for signing the payload.
+  ::
+  ::field{name="provider" type="null | Provider"}
+  The provider is used to fetch data from the network if it is required for signing.
+  ::
+::
 
 ## `PriorityOpResponse`
 
-Interface representation of priority op response that extends[`TransactionResponse`](#transactionresponse)
-and adds a function that waits to commit a layer 1 transaction, including when given on optional confirmation number.
+Interface representation of priority op response that extends [`TransactionResponse`](#transactionresponse)
+and adds a function that waits to commit a layer 1 transaction, including when given an optional confirmation number.
 
-- `waitL1Commit(confirmation?: number)`: `Promise<providers.TransactionReceipt>` - Waits for the L1 transaction to be committed,
-including waiting for the specified number of confirmations.
+::collapsible
+  ::field{name="waitL1Commit" type="function"}
+  Waits for the L1 transaction to be committed, including waiting for the specified number of confirmations.
+  ::
+::
 
 ## `PriorityOpTree`
 
 Enumerated list of priority operation tree types.
 
-- Full = `0`,
-- Rollup = `1`,
+::collapsible
+  ::field{name="Full" type="number" defaultValue="0"}
+  ::
+  ::field{name="Rollup" type="number" defaultValue="1"}
+  ::
+::
 
 ## `PriorityQueueType`
 
 Enumerated list of priority queue types.
 
-- Deque = `0`,
-- HeapBuffer = `1`,
-- Heap = `2`,
+::collapsible
+  ::field{name="Deque" type="number" defaultValue="0"}
+  ::
+  ::field{name="HeapBuffer" type="number" defaultValue="1"}
+  ::
+  ::field{name="Heap" type="number" defaultValue="2"}
+  ::
+::
```

Comment:
Again, not separate properties. Please check through each section, I did not go through each one to ensure they were all correct.